config/crd/bases/dynatrace.com_edgeconnects.yaml

```
@@ -207,6 +207,13 @@ spec:
                     description: Indicates version of the EdgeConnect image to use
                     type: string
                 type: object
+              kubernetesApiAutomation:
```

There was a small change requested, please change it to
```suggestion
              kubernetesAutomation:
```

---

[Renamed](https://github.com/Dynatrace/dynatrace-operator/pull/2977/commits/e2669335c066680be16daa0681c9f34f6563395b)